optimum/exporters/onnx/model_configs.py

```diff
@@ -751,6 +751,20 @@ class WhisperOnnxConfig(AudioToTextOnnxConfig):
     NORMALIZED_CONFIG_CLASS = NormalizedSeq2SeqConfig
     ATOL_FOR_VALIDATION = 1e-3
 
+    def _create_dummy_input_generator_classes(self, **kwargs) -> List["DummyInputGenerator"]:
+        dummy_inputs_generators = super()._create_dummy_input_generator_classes(**kwargs)
+
+        if self._behavior is ConfigBehavior.MONOLITH:
```

Comment:
```suggestion
        # in the monolith case, we need to hint that the sequence length is halved in the decoder
        if self._behavior is ConfigBehavior.MONOLITH:
```

or something like this

Reply:
Added a comment